src/view/popup-view.js

```diff
@@ -1,135 +1,138 @@
 import {createElement} from '../render.js';
+import {humanizeDate, humanizeFilmRuntime, sortComments} from '../utils.js';
 
-const createPopup = () => (
-  `<section class="film-details">
+const POPUP_RELEASE_DATE_FORMAT = 'DD MMMM YYYY';
+const POPUP_COMMENT_DATE_FORMAT = 'YYYY/MM/DD h:mm';
+
+const updateGenreTerm = (currentGenres) => (currentGenres.length > 1) ? 'Genres' : 'Genre';
+
+const createGenreElements = (currentGenres) => {
+  let genresCollection = '';
+
+  currentGenres.forEach((item) => {
+    genresCollection += `<span class="film-details__genre">${item}</span>`;
+  });
+
+  return genresCollection;
+};
+
+const createCommentsList = (comments) => {
+  let commentsList = '';
+
+  comments.forEach(({author, comment, date, emotion}) => {
+
+    const commentDate = humanizeDate(date, POPUP_COMMENT_DATE_FORMAT);
+
+    commentsList += (
+      `<li class="film-details__comment">
+      <span class="film-details__comment-emoji">
+        <img src="./images/emoji/${emotion}.png" width="55" height="55" alt="emoji-smile">
+      </span>
+      <div>
+        <p class="film-details__comment-text">${comment}</p>
+        <p class="film-details__comment-info">
+          <span class="film-details__comment-author">${author}</span>
+          <span class="film-details__comment-day">${commentDate}</span>
+          <button class="film-details__comment-delete">Delete</button>
+        </p>
+      </div>
+      </li>`);
+  });
+
+  return commentsList;
+};
```

Comment:
Правильнее использовать метод map
```suggestion
const createCommentsList = (comments) =>
  comments.map(({author, comment, date, emotion}) => (
    `<li class="film-details__comment">
      <span class="film-details__comment-emoji">
        <img src="./images/emoji/${emotion}.png" width="55" height="55" alt="emoji-smile">
      </span>
      <div>
        <p class="film-details__comment-text">${comment}</p>
        <p class="film-details__comment-info">
          <span class="film-details__comment-author">${author}</span>
          <span class="film-details__comment-day">${humanizeDate(date, POPUP_COMMENT_DATE_FORMAT)}</span>
          <button class="film-details__comment-delete">Delete</button>
        </p>
      </div>
      </li>`
  )).join('');
```